src/main/java/com/example/demo/controller/CarController.java

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.controller;
+
+import com.example.demo.ResponseStatus.ResponseStatus;
+import com.example.demo.exceptions.InvalidDataException;
+import com.example.demo.exceptions.ObjectAlreadyExistsException;
+import com.example.demo.exceptions.ObjectNotFoundException;
+import com.example.demo.model.Car;
+import com.example.demo.service.CarService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+
+@RestController
+public class CarController {
+
+    @Autowired
+    private CarService carService;
+
+    @RequestMapping(method= RequestMethod.POST,value="/newcar")
```

Comment:
to jest poprawne ale jest coś takiego jak @PostMapping co wg mnie jest czytelniejsze